orchagent/portsorch.cpp

```diff
@@ -2013,16 +2013,8 @@ bool PortsOrch::setPortAdvInterfaceTypes(sai_object_id_t port_id, std::vector<ui
     attr.value.u32list.count = static_cast<uint32_t>(interface_types.size());
 
     status = sai_port_api->set_port_attribute(port_id, &attr);
-    if (status != SAI_STATUS_SUCCESS)
```

Comment:
This change removes the failure notification completely. I believe this is not the expected behavior. According to your issue description, it should only be skipped if the returned status is `SAI_STATUS_INVALID_ATTR_VALUE_0`. Therefore, I think the change here should be either checking for status before this part like https://github.com/Azure/sonic-swss/blob/master/orchagent/tunneldecaporch.cpp#L760, or add is as a specific case in the failure handling function like https://github.com/Azure/sonic-swss/pull/1815/files